src/content/contribute/plugin-patterns.md

```diff
@@ -48,21 +49,21 @@ module.exports = MyPlugin;
 
 ### Monitoring the watch graph
 
-While running webpack middleware, each compilation includes a `fileDependencies` array (what files are being watched) and a `fileTimestamps` hash that maps watched file paths to a timestamp. These are extremely useful for detecting what files have changed within the compilation:
+While running webpack middleware, each compilation includes a `fileDependencies` set (what files are being watched) and a `fileTimestamps` map that maps watched file paths to a timestamp. These are extremely useful for detecting what files have changed within the compilation:
```

Comment:
It would be better to say explicitly that it is a JavaScript Set and Map.

Reply:
Hmm true, want to trow a PR or should i take on it @montogeek ?

Reply:
Feel free to do it :)